visallo/visallo
# fix possible values not getting populated from saved search

Testing Instructions: 
1) Create a saved search with gender: Male
2) Run the saved search
3) Search filters should be populated with Gender Male

Points of Regression:

NO CHANGELOG
